Index.html

```diff
@@ -0,0 +1,303 @@
+        <section class="nav-items">
+          <ul>
+            <li>Destinations</li>
+            <li>Hotels</li>
+            <li>Flights</li>
+            <li>Bookings</li>
+            <li>Login</li>
+            <div class="signup-btn">Sign up</div>
+            <div class="language-dropdown">
+              <span class="dropdown-trigger">EN&#9660;</span>
+              <ul class="dropdown-content">
+                <li>Hindi</li>
+                <li>German</li>
+              </ul>
+            </div>
+          </ul>
+        </section>
+      </nav>
+      <main>
+        <div class="landing-left-section">
+          <h4>BEST DESTINATIONS AROUND THE WORLD</h4>
+          <h1>
+            Travel, enjoy <br />and live a new <br />
+            and full life
+          </h1>
+          <p>
+            Built Wicket Longer admire do barton vanity itself do in it. <br />
+            Preferred to sportsmen it engrossed listening. Park gate <br />
+            sell they west hard for the.
+          </p>
+          <button class="cta-button">Find out more</button>
+        </div>
+        <div class="landing-right-section">
+          <img src="images\traveller.png" alt="traveller" />
+        </div>
+      </main>
+    </header>
+    <section class="category">
+      <div>
+        <div class="category">CATEGORY</div>
+        <section class="category-content">We Offer Best Services</section>
+      </div>
+      <img src="images\plus.png" alt="plus" />
+    </section>
+
+    <div class="services-list">
+      <div class="service-card">
+        <div class="card-img">
+          <img src="images\weather.png" alt="Weather Image" />
+        </div>
+
+        <div class="card-content">
+          <h2>Calculated Weather</h2>
+          <p>Built Wicket longer admire do barton vanity itself do in it.</p>
+        </div>
+      </div>
+      <div class="service-card">
+        <div class="card-img">
+          <img src="images\flights.png" alt="Flight Image" />
+        </div>
+        <div class="card-content">
+          <h2>Best Flights</h2>
+          <p>Engrossed listening. Park gate sell they west hard for the.</p>
+        </div>
+      </div>
+      <div class="service-card">
+        <div class="card-img">
+          <img src="images\event.png" alt="Event Image" />
+        </div>
+        <div class="card-content">
+          <h2>Local Events</h2>
+          <p>
+            Barton vanity itself do in it. Preferred to men it engrossed
+            listening.
+          </p>
+        </div>
+      </div>
+      <div class="service-card">
+        <div class="card-img">
+          <img src="images\customization.png" alt="Customization Image" />
+        </div>
+        <div class="card-content">
+          <h2>Customization</h2>
+          <p>We deliver outsourced aviation services for military customers.</p>
+        </div>
+      </div>
+    </div>
+    <section class="top-selling">
+      <div class="selling-header">

```

Comment:
use `header` tag for defining header in section

Reply:
resolved